force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,11 +68,16 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+     If (num1 == null || num2 == null) {
+        return null;
+     }
+
+     if (num1 > num2) {
+        return num1;
+        }else {
+            return num2;
+        }
```

Comment:
Try using the Math class https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm#apex_System_Math_max